test/integration/fee/Permit2FeeCharging.t.sol

```diff
@@ -37,6 +35,9 @@ contract Permit2FeeCalculatorTest is Test, ERC20Permit2Utils, TypedDataSignature
     bytes32[] public referralsEmpty;
 
     function setUp() external {
+        vm.createSelectFork(vm.rpcUrl('ethereum'));
+        TypedDataSignature.initialize();
```

Comment:
Is `TypedDataSignature` here necessary? Or we can simply use `initialize()`?

Reply:
The main idea is to know what are we going to `initialize` without browsing the code. I also tried another name like `initializeTypeDataSignature`, but it was not as clean as the one we used.

Reply:
Got it, makes sense.